rpcs3/Emu/RSX/Overlays/HomeMenu/overlay_home_menu_main_menu.cpp

```diff
@@ -52,12 +52,12 @@ namespace rsx
 
 				Emu.CallFromMainThread([suspend_mode]()
 				{
-					Emu.Kill(false, true);
-
-					if (!suspend_mode)
+					Emu.after_kill_callback = [suspend_mode]()
```

Comment:
why capture suspend_mode when you don't use it ?

Reply:
Fixed.